auto_post_classifier/utils.py

```diff
@@ -4,6 +4,8 @@
 import openai
 import pandas as pd
 
+from loguru import logger
```

Comment:
@nilnil47 seems like you're not using the logger in the `utils`. If so, please remove it.